exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -87,14 +77,31 @@ processors:
         # detect cluster name and location
         detectors: [gcp]
         timeout: 10s
+    transform:
+      metric_statements:
```

Comment:
```suggestion
      # "location", "cluster", "namespace", "job", "instance", and "project_id" are reserved, and 
      # metrics containing these labels will be rejected.  Prefix them with exported_ to prevent this.
      metric_statements:
```